### DISCUSSION TITLE
Thank you for this project

### DISCUSSION THREAD

**Discussion by chewbakartik** [input-leap/input-leap#1521] [General]
I couldn't find anywhere else to add this, so I thought adding a new discussion here would make the most sense, hopefully the devs maintaining this project get a chance to see this. Thanks for all the hard work that you do. As a long time Synergy user, I never knew this fork (and barrier) existed.

I had been getting increasingly frustrated with Synergy over the course of the last 6-12 months, but as my workflow depended on the functionality, I pushed through until recently. The major bugs that I've experienced using Synergy with a linux host, osx client include the shift & win/cmd key not working on the osx client when that machine has used RDP to a Windows server. Most recently, there's been a problem where my host machine after a while of having Synergy running the monitors refuse to dim and the mouse primary click starts to lag/delay.

This project has worked for me, right out of the box, much like the classic Synergy used to and I very much appreciate it. Thanks for keeping things simple and dedicated to the primary function of what is needed.

**Comment by shymega** [comment]
Thank you. I'm trying to learn C++ to speed-up development, but also experimenting with Rust to help mitiigate some long-term issues, and UTF-8. This isn't official, but it _is_ on a branch on this repo. It's more of my hobby project.

**Comment by A-Metal-Penguin** [comment]
Just wanted to say thank you as well! This has been a wonderful tool to use on my systems. Seriously! Such a time saver and resolves so much frustration!!!
